### ISSUE TITLE
[Disscussion]: Remove 'squashed commits' requirement

### ISSUE THREAD

**Issue by Bouncey** [freeCodeCamp/freeCodeCamp#16732]
Now with `Squash and merge`, should we remove all references in the docs and Camperbot about squashing commits?

One reason for is, after reviewing a PR and requesting changes, we lose review comments made against a commit because it doesn't exist any more. This makes checking that requested changes were made more of a memory game.

**Comment by raisedadead** [comment]
Yes the GitHub bot needs some love, which I intend to do.

**Comment by QuincyLarson** [comment]
@raisedadead @Bouncey OK - I'm adding this to the road map. Super low priority, but this is still something we should do imho.

**Comment by manzilo** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  manzil@gmail.com
    Domain funtasticlabs.com has exceeded the max emails per hour (114/100 (113%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from mailin-04.elitehosts.com ([72.9.130.158]:39828)
	by neo.internetwebserver.net with esmtp (Exim 4.87)
	(envelope-from <noreply@github.com>)
	id 1fPUQD-0007TB-Qk
	for manzil@funtasticlabs.com; Sun, 03 Jun 2018 10:55:53 -0400
Received: from out-5.smtp.github.com ([192.30.252.196])
	by mailin-04.elitehosts.com with esmtp (Exim 4.69)
	(envelope-from <noreply@github.com>)
	id 1fPUPH-0006Jn-SU
	for manzil@funtasticlabs.com; Sun, 03 Jun 2018 10:54:56 -0400
Date: Sun, 03 Jun 2018 07:54:54 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1528037695;
	bh=W0dkWNcqT5wYJ9MoJev1QRtvCT/t2ZPnLaKpITtfwFw=;
	h=Date:From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=ic/1jHnk0bPNQ6wB70Grb+9R9ODNjVStOdma92UtPdZdzgqwCbUFVkMvNVZ709tve
	 MiOzbxdhqZLRDNm2IMdYwxpOn88e2nWrCJEMdv0qKk5zDl4CCj9W64WAjfQfRUoCpw
	 QOskFwy/WtoF1GqZKwutcvF+/cYxpDLzJXclPLMk=

**Comment by Vimal-Raghubir** [comment]
Hi, is it possible for me to tackle this issue?

**Comment by QuincyLarson** [comment]
@Vimal-Raghubir yes - we'd welcome your help with it. Please keep us posted on how you progress with it, and let us know if we can be of any help.

**Comment by raisedadead** [comment]
Hi @Vimal-Raghubir @QuincyLarson we are planning to move away to https://probot.github.io instead of the current outdated bot. This is best to fix up with the re-write I am planning.

**Comment by Vimal-Raghubir** [comment]
So this is something that you will be adressing in the new robot's setup @raisedadead? Therefore it will be redundant if I start digging into this issue for now. If my understanding is correct.

**Comment by raisedadead** [comment]
That is correct @Vimal-Raghubir You are more than welcome to take a look at https://github.com/freeCodeCamp/camper-gh-bot and wait for my progress there, and then on I would love your help on some of the user stories if you would be interested.

**Comment by Vimal-Raghubir** [comment]
I would definitely be interested in helping out with the user stories. @raisedadead
